middleware/logger.go

```diff
@@ -0,0 +1,13 @@
+package middleware
+
+import (
+	"net/http"
+
+	"github.com/speedrun-website/leaderboard-backend/logger"
+)
+
+// Make a logging middleware handler with the configured logger injected.
+func LoggingMiddleware(rw http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
+	logger.Logger.Info().Msgf("Handling Request: %s %s %s", r.RemoteAddr, r.URL, r.Method)
```

Comment:
How do we make sure that `logger.Logger` is defined at the point of calling this?

Reply:
We don't necessarily make sure that `InitLogger` has been run. If I remember correctly though (it's been a while since I looked at this) if InitLogger hasn't been run it doesn't break or anything. It just won't have the settings we want; it will have the settings of the 0-value `zerolog.Logger`.